app/footer.tsx

```diff
@@ -1,3 +1,222 @@
-export default function Footer() {
-  return <></>;
+// import React from 'react'
+import Image from "next/image";
+import footerimg from "../components/assets/footerimg.png";
+import twitter from "../components/assets/twitter.svg";
+import fb from "../components/assets/fb.svg";
+import linkedin from "../components/assets/linkedin.svg";
+import logo from "../components/assets/nitkkrlogo.png";
+import call from "../components/assets/call.svg";
+import mail from "../components/assets/mail.svg";
+import distance from "../components/assets/distance.svg";
+import lang from "../components/assets/language.svg";
+
+
+const quickLinks = [
+  {
+    text: "Quick Links",
+    style: "font-black md:text-[24px] py-3 sm:text-[16px]",
+  },
+  {
+    text: "Telephone Directory",
+    style: "font-medium text-[#8B949E] md:text-[20px] py-1 sm:text-[12px]",
+  },
+  {
+    text: "Books and e-library",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
+  },
+  {
+    text: "Important Links",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
+  },
+  {
+    text: "Downloads",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
+  },
+  {
+    text: "RTI",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
+  },
+  {
+    text: "NIT KKR IT Infrastructure Usage Policy",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
+  },
+  {
+    text: "NIT KKR @NDL",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
+  },
+  {
+    text: "Telephone Directory",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
+  },
+];
+
+
+
+const aboutUsItems = [
+  { text: "About Us", style: "font-bold md:text-[24px] py-3 sm:text-[16px]" },
+  {
+    text: "Council of NITs",
+    style: "font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]",
```

Comment:
It will be `text` and `link`. Style is common to all and won't be included in the object